src/containers/settings/EditServiceScreen.tsx

```diff
@@ -468,6 +468,16 @@ class EditServiceScreen extends Component<IProps> {
     }
   }
 
+  clearCache(): void {
+    const { action } = this.props.params;
+
+    if (action === 'edit') {
+      const { clearCache } = this.props.actions.service;
+      const { activeSettings: service } = this.props.stores.services;
+      clearCache(service?.id);
```

Comment:
Shouldn't this be `clearCache({ serviceId: service?.id });` in the same way the other actions work by using an object instead of only a string?

Reply:
thanks @Alphrag - fixed the review comment